Readme.md

```diff
@@ -20,9 +20,9 @@ kubectl create ns kvrocks
 
 3. Use helm to install and manage the crd/operator 
 ```
-git clone https://github.com/RocksLabs/kvrocks-operator.git
-helm install kvrocks-crd deploy/crd -n kvrocks
-helm install kvrocks-operator deploy/operator -n kvrocks
+helm repo add kvrocks-operator https://rockslabs.github.io/kvrocks-operator
+helm install kvrocks-crd kvrocks-operator/kvrocks-crd -n kvrocks
```

Comment:
I believe we can relocate the crd directory to the operator, similar to this example: https://github.com/jiayouxujin/helm-charts/tree/master/charts/kvrocks-operator.